studio/schemas/documents/sections/awards.js

```diff
@@ -0,0 +1,59 @@
+import supportedLanguages from '../../supportedLanguages'
+import { SplitHorizontalIcon } from '@sanity/icons'
+
+const baseLanguage = supportedLanguages.find(l => l.isDefault)
+
+export default {
+  type: 'document',
+  name: 'awards',
+  title: 'Awards',
+  icon: SplitHorizontalIcon,
+  fields: [
+    {
+      name: 'heading',
+      type: 'localeString',
+      title: 'Heading (*)',
+      validation: Rule => Rule.error('Information required.').required()
+    },
+    {
+      name: 'images',
+      type: 'array',
+      title: 'Images (*)',
```

Comment:
Since it is an carousel, it would be good to change the title and add a description:

      title: 'Awards Carousel',
      description 'Choose the images that will be displayed in the scrollable carousel',